src/load.ts

```diff
@@ -82,6 +82,23 @@ export async function load(
     }
 }
 
+/**
+ * Loads the data from a CDN and returns an instance of AzureAppConfiguration.
+ * @param cdnEndpoint  The URL to the CDN.
+ * @param appConfigOptions  Optional parameters.
+ */
+export async function loadFromCdn(endpoint: URL | string, options?: AzureAppConfigurationOptions): Promise<AzureAppConfiguration>;

```

Comment:
> endpoint

Should this be updated too?

Reply:
weird, I remembered I updated this in the commit, will update it again